### DISCUSSION TITLE
Import JSON structure

### DISCUSSION THREAD

**Discussion by tiagobugarin** [beromir/Servas#68] [Ideas]
As Pocket only exports HTML and Servas only imports JSON it would be nice to have a tool to convert from the former to the later.

I can try to make something in that regard in Python. I am not an experienced developer but I can give it a try.  
To do this I think I would need a JSON structure of how Servas would expect to import.

**Comment by beromir** [comment]
Hi,
HTML import and export is planned as browsers use this format as well.
JSON import and export is more for backing up the data and migrating to another instance.
